src/shadow-embed.js

```diff
@@ -61,7 +62,10 @@ export function createShadowRoot(hostElement) {
   // Native support.
   const shadowDomSupported = getShadowDomSupportedVersion();
   if (shadowDomSupported == ShadowDomVersion.V1) {
-    return hostElement.attachShadow({mode: 'open'});
+    // NOTE(torch2424) need to have an id on the shadow root no matter what
+    const shadowRoot = hostElement.attachShadow({mode: 'open'});
```

Comment:
(a) I think something like this may be needed for V0 and V1 both; (b) I don't think it'll help to assign this to the `DocumentFragment` as in the ShadyDOM case.